src/store/channels-list/api.ts

```diff
@@ -16,3 +16,11 @@ export async function createConversation(userIds: string[]): Promise<DirectMessa
   const directMessages = await post<Channel[]>('/directMessages').send({ userIds });
   return directMessages.body;
 }
+
+export async function fetchConversationsWithUsers(_userIds: string[]): Promise<any[]> {
+  // Simulate request timing
```

Comment:
This looks like test / debug code?

Reply:
Yep, this is intentional because the endpoint doesn't exist yet. For now, the system behaves exactly the same as it did previously where it doesn't check for existing conversations. Once the endpoint exists this can be enabled.